scripts/forest_inventory_analysis.py

```diff
@@ -16,14 +16,14 @@
 class main(Script):
     def __init__(self, **kwargs):
         Script.__init__(self, **kwargs)
-        self.name = "Forest Inventory and Analysis"
-        self.shortname = "forest-inventory-analysis"
+        self.title = "Forest Inventory and Analysis"
+        self.name = "forest-inventory-analysis"
         self.retriever_minimum_version = '2.0.dev'
-        self.version = '1.3.1'
+        self.version = '1.4.0'
         self.ref = "http://fia.fs.fed.us/"
         self.urls = {"main": "https://apps.fs.usda.gov/fiadb-downloads/CSV/",
                      'species': 'https://apps.fs.usda.gov/fiadb-downloads/CSV/REF_SPECIES.csv'}
-        self.tags = ["plants", "continental-scale", "observational"]
+        self.keywords = ["plants", "continental-scale", "observational"]
```

Comment:
@henrykironde I have made some changes in the script `MammalSuperTree.py`, it doesn't look much clean. I had put in some thought, unfortunately couldn't come with something better. Kindly have a look and suggest if something better comes up.